index.html

```diff
@@ -32,15 +32,16 @@
             class="navigation_bar__link link size-20 weight-700 line_hieght-20 bar_head"
             >GERGITO</a
           >
+          <span class="close-btn non-visible">X</span>
           <img
             class="navigation_bar__hamb hamburguer-img"
             src="img/hamburguer.png"
             alt="hamburguer icon"
           />
-          <ul class="navigation_bar menu-list">
-            <li><a href="#works" >Portfolio</a> </li>
-            <li><a href="#about" >About</a></li>
-            <li><a href="#contact-me" >Contact</a></li>
+          <ul class="menu-list non-visible">
+            <li><a class="li-a" href="#works" >Portfolio</a> </li>
+            <li><a class="li-a2" href="#about" >About</a></li>
+            <li><a class="li-a3" href="#contact-me" >Contact</a></li>

```

Comment:
- [ ] Please ensure in the mobile menu, overflow is hidden, it covers the whole page and has a blurry background as in the given figma design:
- figma
![image](https://user-images.githubusercontent.com/91486791/232478244-97f4a74e-6a0e-4d06-985f-a42a87cc8261.png)
- your design
![image](https://user-images.githubusercontent.com/91486791/232478428-ff9b0f51-6d94-4605-8990-e7bdaeae3a4f.png)

Reply:
###  👍 👍 👍  Okay thank you much for your nice and constructive comment. 👍 👍 👍